### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by cahya-wirawan** [devJWSong/gpt2-chatbot-pytorch#5]
Hi, do you have a pretrained model. I would like to test it. Thanks

**Comment by devjwsong** [comment]
Hello. Thank you for visiting this project. I want to ask you that what "a pretrained model" exactly means. If you mean the model which was fine-tuned on dialogue generation, unfortunately I don't have any. If you want to have the literally "pretrained" GPT-2 model, you can access it in Huggingface's Transformers library, which I also adapted in this project. Please let me know if you need additional help. Thank you.

**Comment by cahya-wirawan** [comment]
I mean the model you have fine-tuned on dialogue generation. I am currently fine tune it following your repository, I tested the checkpoint from epoch 4, but the result is not really good. So I am wondering if you have better experience with the training/fine tuning. Thanks

**Comment by devjwsong** [comment]
I see. I'm afraid that I don't have any trained checkpoint. But in my experience, I could get fine results after 20 epochs. So I think you should train the model more longer, or adjust learning rate by changing it or adding a learning rate scheduler.
